### PR TITLE
Feat/paper list add subheader by contact

### PR DESCRIPTION
**Pull request by Merkur39**
See commits messages

### CODE REVIEW

File: src/utils/referencedBy.js [hunk 1]
```diff
@@ -5,14 +5,20 @@
...
+export const isReferencedBy = (file, referencedBy, id) => {
+  const references =
+    file?.relationships?.referenced_by?.data || file?.referenced_by || []
+
+  if (!id) {
     return !!references.find(reference => reference.type === referencedBy)
   }
-  return false
+
+  return !!references.find(
```
Comment (Ldoppea) [inline src/utils/referencedBy.js:19]: Maybe you can replace `!!references.find(...)` by `references.some(...)`

File: src/helpers/queries.js [hunk 2]
```diff
@@ -53,3 +53,11 @@ export const getOnboardingStatus = {
     fetchPolicy: fetchPolicies.noFetch()
   }
 }
+
+export const getContactByIds = (ids = []) => ({
+  definition: () => Q(CONTACTS_DOCTYPE).getByIds(ids),
+  options: {
+    as: `getContactByIds/${ids.join('')}`,
```
Comment (Ldoppea) [inline src/helpers/queries.js:60]: :question: How does this work if there is no seperator between id strings?

**Review by Ldoppea — APPROVED** [review]

**Comment by Merkur39** [comment]
[RG25 - Gestion séparateurs par user](https://trello.com/c/DU5QAEtT/796-rg25-gestion-s%C3%A9parateurs-par-user)

File: src/helpers/queries.js (hunk 2 above)
Comment (Merkur39) [reply src/helpers/queries.js:60]: The IDs will be stuck together, which is fine in this case

File: src/utils/referencedBy.js (hunk 1 above)
Comment (Merkur39) [reply src/utils/referencedBy.js:19]: Helpers will be implemented in [cozy-client](https://github.com/cozy/cozy-client/releases/tag/v27.5.1)] now!
So I will delete this file and update cozy-client package in the dependencies.
PS: I made the changes on https://github.com/cozy/cozy-client/pull/1076.

**Pull request merged**